docs/modules/gffcompare.md

```diff
@@ -0,0 +1,12 @@
+---
+Name: GffCompare
+URL: https://ccb.jhu.edu/software/stringtie/gffcompare.shtml
+Description: >
+    is a tool to compare, merge and annotate one or more GFF files with a reference annotation in GFF format.
+---
+
+Currently, nothing is added to the general statistics table. Thisis ebcause there is not a single most important key value. Additionally this allows multiple GffCompare results for the same sample (different references?) having a different name without bloating the general stats table.
+
+The Sensitivity/Precision values are displayed in a single plot, different loci levels can be switched by choosing a different dataset.
+
+Please use gffcompare only with single samples. Multi-Sample comparisons are not correctly rendered by this module.
```

Comment:
I don't love this. It would be much better if we could skip multi-sample comparisons with a warning, I don't like that it renders incorrectly.

Would be good to add some example data to the TestData repo for this too.